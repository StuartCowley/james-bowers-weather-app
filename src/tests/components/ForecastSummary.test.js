import React from "react";

describe("ForecastSummmary", () => {
  it("renders", () => {});
});
