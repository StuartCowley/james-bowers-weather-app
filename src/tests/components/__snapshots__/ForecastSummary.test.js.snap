// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastSummmary renders correctly 1`] = `
<DocumentFragment>
  <div
    class="forecast-summary"
    data-testid="forecast-summary"
  >
    <div
      class="forecast-summary__date"
    >
      Fri Mar 18
    </div>
    <div
      class="forecast-summary__icon"
      data-testid="forecast-icon"
    >
      <i
        class="wi wi-day-sunny"
        name="owm"
      />
    </div>
    <div
      class="forecast-summary__maxTemp"
    >
      <p
        class="forecast-summary__maxTemp"
      >
        22Â°C
      </p>
    </div>
    <div
      class="forecast-summary__description"
    >
      Stub description
    </div>
    <button
      type="button"
    >
      More details
    </button>
  </div>
</DocumentFragment>
`;
